lambdas/getSunriseSunset.mjs

```diff
@@ -14,69 +10,58 @@
 //     "tomorrowSunrise": "2024-10-01T07:04:41.000Z"
 // }
 
-export const handler = async (event) => {
-  let lat, lng;
+export const handler = async function (event) {
+  console.log("Received event:", JSON.stringify(event, null, 2));
 
-  if (!event.body) {
-    // Works in the AWS test environment
-    lat = event.lat;
-    lng = event.lng;
-  } else {
-    // Works for frontend
-    const body = JSON.parse(event.body);
-    lat = body.lat;
-    lng = body.lng;
-  }
+  let latitude, longitude;
 
-  // Error if lat, lng is not passed in
-  if (!lat || !lng) {
-    return {
-      statusCode: 400,
-      body: JSON.stringify({
-        error: "Latitude and longitude are required",
-        eventBody: event.body,
-      }),
-    };
+  // Extract latitude and longitude from the event
+  if (event.latitude && event.latitude !== "" && event.longitude && event.longitude !== "") {
```

Comment:
Empty string is falsy
https://developer.mozilla.org/en-US/docs/Glossary/Falsy

I think `if (event.latitude && event.longitude)` is enough